src/rain.c

```diff
@@ -0,0 +1,469 @@
+/*
+ * This software is Copyright (c) 2018 magnum
```

Comment:
I did not write this, please remove my name from it and put your own there

Reply:
magnum, since your Subsets mode code was used as a template, I guess this is derived work and needs to list both of you as copyright holders, unless the remnants of your code are so insignificant they're not subject to copyright. Separately, a comment may be added that you don't endorse this mode, or something. ;-)

It doesn't look likely that we'd accept this, but even if it exists in a fork of jumbo only, it should better have proper copyright statements and disclaimer.

Reply:
I think the "template" was merely for seeing how fix/save/restore_state and such things are implemented so I really don't want to be mentioned at all.